modules/op-running-a-pipeline.adoc

```diff
@@ -14,7 +14,7 @@ A `PipelineRun` resource starts a pipeline and ties it to the Git and image reso
 +
 ----
 $ tkn pipeline start build-and-deploy \
-    -w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/01_pipeline/03_persistent_volume_claim.yaml \
+    -w name=shared-workspace,volumeClaimTemplateFile=https://github.com/openshift/pipelines-tutorial/blob/release-tech-preview-3/01_pipeline/03_persistent_volume_claim.yaml \
```

Comment:
You need to revert back to `raw.githubusercontent.com`, this URL would not work properly.